### ISSUE TITLE
samples missing from SDK 1.2.0

### ISSUE THREAD

**Issue by handcyclist** [Azure/azure-iot-sdk-c#406]
The following samples are missing from SDK 1.2.0:

iothub_client_sample_mqtt
iothub_client_sample_http
iothub_client_sample_amqp

I have SDK version 1.1.29 from Jan 2, 2018 and it included these samples.
I have done more installs to work with Ubuntu 14.04 and Raspberry Pi 3 and realized that these samples are missing in SDK 1.2.0

The ..._mqtt is the sample referred to in the documentation:
https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-sdk-c-intro

The ..._http & ..._amqp are the samples referred to in  the documentation:
https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-sdk-c-iothubclient

**Comment by jspaith** [comment]
You're correct, we've removed these samples.  They were problematic:
* Because there were so many lines of {..} in them they were hard to follow.
* They gave appearance of being ready to productize, which they weren't really.
* There was **massive** copy/paste between them; pretty much protocol variable was only difference.

We've created a new sample in the same directory called `samples\iothub_ll_telemetry_sample`.  Since you're familiar with the existing ones, we'd definitely take any additional feedback you have on this.

The fact that the docs are referencing the old samples is a good catch and we'll definitely address that.  Thanks
